addons/mail/static/src/components/composer/composer.xml

```diff
@@ -163,7 +163,7 @@
                     type="button"
                     t-on-click="_onClickSend"
                 >
-                    <t t-if="!messaging.device.isMobile"><t t-if="composerView.composer.isLog">Log</t><t t-else="">Send</t></t>
+                    <t t-if="!messaging.device.isMobile"><t t-if="composerView.composer.isLog and composerView.composer.activeThread and composerView.composer.activeThread.model !== 'mail.channel'">Log</t><t t-else="">Send</t></t>
```

Comment:
Maybe it's time to add `sendButtonText` field in composer view instead of complex code in the template?